internal/app/mapper/mapper.go

```diff
@@ -227,15 +297,25 @@ func getResultFromRequestNodeFragment(fragmentRule *rule, node *core.Node) (bool
 	case aggregationv1.NodeFieldType_NODE_LOCALITY_SUBZONE:
 		nodeValue = node.GetLocality().GetSubZone()
 	default:
-		return false, "", fmt.Errorf("RequestNodeFragment Invalid NodeFieldType")
+		return "", fmt.Errorf("RequestNodeFragment Invalid NodeFieldType")
 	}
 
+	return nodeValue, nil
+}
+
+func getResultFragmentFromAction(
+	requestNodeFragment *resultPredicateRequestNodeFragment,
+	node *core.Node) (string, error) {
```

Comment:
```suggestion
func getResultFragmentFromAction(
	requestNodeFragment *resultPredicateRequestNodeFragment,
	node *core.Node,
) (string, error) {
```

Reply:
It can probably be all on one line once you shorten the `resultPredicateRequestNodeFragment` name.